packages/grafana-ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -69,6 +69,7 @@ function getStyles(theme: GrafanaTheme2) {
       transition: opacity 0.3s;
       z-index: ${theme.zIndex.tooltip};
       max-width: 400px;
+      overflow-wrap: break-word;
```

Comment:
This will prevent long words (e.g. in json object where there are no spaces) to overflow (outside of tooltip background).